### ISSUE TITLE
Android crash: Attempt to invoke virtual method 'android.content.res.Resources android.content.Context.getResources()' on a null object reference

### ISSUE THREAD

**Issue by jsierles** [tipsi/tipsi-stripe#90]
It looks like the library tries to show an error. Here's the backtrace:

```
Toast.java:138android.widget.Toast.<init>	
Toast.java:385android.widget.Toast.makeText	
AddCardDialogFragment.java:210com.gettipsi.stripe.dialog.AddCardDialogFragment$4.onError	
Stripe.java:341com.stripe.android.Stripe.tokenTaskPostExecution	
Stripe.java:32com.stripe.android.Stripe.access$100	
Stripe.java:60com.stripe.android.Stripe$1$1.onPostExecute	
Stripe.java:43com.stripe.android.Stripe$1$1.onPostExecute	
AsyncTask.java:660android.os.AsyncTask.finish	
AsyncTask.java:-1android.os.AsyncTask.-wrap1	
AsyncTask.java:677android.os.AsyncTask$InternalHandler.handleMessage	
Handler.java:102android.os.Handler.dispatchMessage	
Looper.java:154android.os.Looper.loop	
ActivityThread.java:6776android.app.ActivityThread.main	
Method.java:-2java.lang.reflect.Method.invoke	
ZygoteInit.java:1520com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run	
ZygoteInit.java:1410com.android.internal.os.ZygoteInit.main
```

Here's the line in question: https://github.com/spin-org/tipsi-stripe/blob/master/android/src/main/java/com/gettipsi/stripe/dialog/AddCardDialogFragment.java#L210

**Comment by isnifer** [comment]
@jsierles Hi, first of all, you provide link to fork, not on our repo. 
Please provide steps to reproduce this error.

**Comment by jsierles** [comment]
Sorry about that. I don't know yet how to reproduce it - it's an exception report from an app in production. I'll post more details when I have them.

**Comment by paynd** [comment]
Hi @jsierles!
Looks like activity was dead at the moment when we trying to show the toast. 
How often does it happens? Can you look into statistics (fabric, crashlitycs?) of this issue?

**Comment by paynd** [comment]
@jsierles Looks like accidental collision. I'll close issue.

**Comment by nikolaik** [comment]
Just saw the same error in our app in prod, pointing to [line 210 in AddCardDialogFragment.java](https://github.com/tipsi/tipsi-stripe/blob/master/android/src/main/java/com/gettipsi/stripe/dialog/AddCardDialogFragment.java#L210).

The exception happened right after the user canceled the dialog. Should we just check if `getActivity()` is not `NULL` before calling `Toast.makeText`?

**Comment by paynd** [comment]
@nikolaik  ok, just created #172

**Comment by isnifer** [comment]
Fixed in 4.4.0

**Comment by Alexei-Suzdalenko** [comment]
private fun  bitmapDescriptorFromVector(context: Activity, vectorResId: Int):BitmapDescriptor {
        var vectorDrawable = ContextCompat.getDrawable(context, vectorResId);
        vectorDrawable!!.setBounds(
            0,
            0,
            vectorDrawable.getIntrinsicWidth(),
            vectorDrawable.getIntrinsicHeight()
        );
        var bitmap = Bitmap.createBitmap(
            vectorDrawable.getIntrinsicWidth(),
            vectorDrawable.getIntrinsicHeight(),
            Bitmap.Config.ARGB_8888
        );
        var canvas =  Canvas(bitmap);
        vectorDrawable.draw(canvas);
        return BitmapDescriptorFactory.fromBitmap(bitmap);
    }
